src/ServiceControl.AcceptanceTesting/Support/ScenarioConfigSource.cs

```diff
@@ -24,9 +24,22 @@ public T GetConfiguration<T>() where T : class, new()
             object configurationSection;
             if (configuration.UserDefinedConfigSections.TryGetValue(type, out configurationSection))
             {
+                if (type == typeof(TransportConfig))
+                {
+                    ((TransportConfig) configurationSection).MaximumConcurrencyLevel = 2;
+                }
+
                 return configurationSection as T;
             }
 
+            if (type == typeof(TransportConfig))
+            {
+                return new TransportConfig
+                {
+                    MaxRetries = 1,
+                    MaximumConcurrencyLevel = 2
```

Comment:
Why these numbers specifically? Should we have a comment or something here explaining?